staging/src/k8s.io/kube-aggregator/pkg/apiserver/handler_discovery.go

```diff
@@ -350,12 +343,8 @@ func (dm *discoveryManager) syncAPIService(apiServiceName string) error {
 	}
 
 	// Lookup last cached result for this apiservice's service.
-	now := time.Now()
 	cached, err := dm.fetchFreshDiscoveryForService(mgv, info)
 
-	info.lastReconciled = now
-	dm.setInfoForAPIService(apiServiceName, &info)
```

Comment:
problematic line.

if apiservice was removed (for instance during long-running fetchFreshDiscoveryForService) then this would then add it back erroneously

Reply:
Is lastReconciled ever set now?

Reply:
It has been removed. It was only used for tests and replaced by `AllAPIServicesSynced`

Reply:
ah, great.